lib/crowbar/client/command/node/list.rb

```diff
@@ -73,6 +73,10 @@ def content_from(request)
                 )
               end
             end
+
+            result.sort do |x, y|
+              x.values.join <=> y.values.join
```

Comment:
Hmm, is this really reliable?  AFAIK the order returned by `Hash#values` is not deterministic, so how do you know whether you are sorting first by name and second by alias, or the other way around?